examples/timed_region.cpp

```diff
@@ -46,6 +47,7 @@ int main(int argc, char**argv)
     iterations = 0;
     while (iterations < total_iterations) {
         geopm_tprof_post();
+        ++iterations;
     }
     geopm_prof_exit(region_id[2]);
 
```

Comment:
Not really part of this review, but worth pointing out:

I see this block of code repeated verbatim three times in this file. It might be worth factoring it out (especially if it is more "generalizable"). If it had been, the fix would've had to be made in only one place.

Reply:
Probably worth considering the removal of the "examples" directory for 2.0.  It is kind of a mess of random use cases.